geomesa-accumulo/geomesa-accumulo-datastore/src/main/scala/org/locationtech/geomesa/accumulo/index/AttributeIdxStrategy.scala

```diff
@@ -72,17 +65,17 @@ class AttributeIdxStrategy(val filter: QueryFilter) extends Strategy with LazyLo
     val priority = FILTERING_ITER_PRIORITY
 
     // query against the attribute table
-    val singleAttrValueOnlyPlan: ScanPlanFn = (schema, filter, transform) => {
-      val iters = KryoLazyFilterTransformIterator.configure(schema, filter, transform, sampling).toSeq
+    val singleAttrValueOnlyPlan: ScanPlanFn = (schema, ecql, transform) => {
```

Comment:
Between the variable names 'filter' and 'ecql', are you using 'ecql' to mean that we have text worth a 'filter'?